faqs/galaxy/datasets_download_datasets.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Download datasets
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+1. Opening the datasets found under history
+2. Clicking on the save icon {% icon galaxy-save %} inside the dataset
```

Comment:
```suggestion
2. Clicking on the save icon {% icon galaxy-save %} inside the dataset.
```

Reply:
I think that it is necessary to keep some consistency regarding the dots at the end of the sentences.